mapping-for-navigation/_posts/0000-01-09-modeling-interesections-for-map-navigation.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Modeling Intersections for Map Navigation
+---
+
+Capturing the various cases of Intersection modelling in OSM and the best practices to follow. The aim of this document is not go into the technicalities of traffic flow in a junction but rather to give a mapper clear cut instructions on how to model different intersections in OSM. Aiming to keep this document simple and easy to follow.
+
+### Case 1: When a transitioning dual carriageway intersects a bi-directional way. It has four ways in which the intersection can be designed 👇 
```

Comment:
It’s confusing that these mapping styles are being presented as “cases”, as though each one is preferred in a different scenario. For more clarity, accompany each diagram with a ✅ or 🚫 symbol indicating whether it’s recommended or not recommended.

Reply:
👍 I found myself getting lost scrolling through several screens of images and loosing track of what the recommended approach is.

Having a consistent pattern that illustrates what the best practice is helps a ton.